app/src/server/trpc/routers/service.ts

```diff
@@ -54,6 +54,14 @@ export const serviceRouter = router({
       return resource as fhir4.Library;
     }),
 
+  getArtifactById: publicProcedure
+    .input(z.object({ resourceType: z.enum(['Measure', 'Library']), id: z.string() }))
+    .query(async ({ input }) => {
+      const res = await fetch(`${process.env.NEXT_PUBLIC_MRS_SERVER}/${input.resourceType}/${input.id}`);
+      const resource = await res.json();
+      return resource as fhir4.Library;
```

Comment:
So I think we want to to return `resource` as `FhirArtifact` (which is a type that we defined that is `Measure | Library`. This is because we are getting an Artifact by id and it can either be a Measure or a Library so we don't want to return it as a library every time. @sarahmcdougall whenever you review this, can you check me on this? Or do we want to specify it is a `fhir4.Library` if `resourceType  = 'Library` and `fhir4.Measure` if `resourceType = 'Measure'` ?

Reply:
I think just returning as a `FhirArtifact` is fine since we never pull off the `resourceType` or any other fields from the resource that we render. Although if the review comments are added to the resource types in different ways then we probably would want to be more specific about the resource type. But I imagine that review comments are added in the same way to both measures and libraries